cmd/webhook/main.go

```diff
@@ -98,17 +99,37 @@ func main() {
 	)
 }
 
+var (
+	_ resourcesemantics.SubResourceLimited = (*crdNoStatusUpdatesOrDeletes)(nil)
+	_ resourcesemantics.VerbLimited        = (*crdNoStatusUpdatesOrDeletes)(nil)
+)
+
+type crdNoStatusUpdatesOrDeletes struct {
+	resourcesemantics.GenericCRD
+}
+
+func (c *crdNoStatusUpdatesOrDeletes) SupportedSubResources() []string {
+	// We do not want any updates that are for status, scale, or anything else.
+	return []string{""}
+}
+
+func (c *crdNoStatusUpdatesOrDeletes) SupportedVerbs() []admissionregistrationv1.OperationType {
+	return []admissionregistrationv1.OperationType{admissionregistrationv1.Create,
+		admissionregistrationv1.Update,
```

Comment:
Can you move Create onto its own line as well?

Reply:
nope